barretenberg/acir_tests/gen_inner_proof_inputs_ultra_honk.sh

```diff
@@ -3,7 +3,7 @@
 #   BIN: to specify a different binary to test with (e.g. bb.js or bb.js-dev).
 set -eu
 
-BIN=${BIN:-../cpp/build/bin/bb}
+BIN=${BIN:-../cpp/build-debug/bin/bb}
```

Comment:
now that the debug build and standard build have diverged, I just changed this to debug, but maybe I can print out something that informs the user that the binary is defaulted to the debug version.